src/klr.core45/tpa.cpp

```diff
@@ -6,78 +6,82 @@
 
 BOOL CreateTpaBase(LPWSTR** ppNames, size_t* pcNames, bool bNative)
 {
-    const size_t count = 32;
+    const size_t count = 34;
```

Comment:
Thanks for the clarification. The update of TPA list is checked in with a separate commit.